packages/bitcoin/build.sh

```diff
@@ -1,10 +1,10 @@
 TERMUX_PKG_HOMEPAGE=https://bitcoincore.org/
 TERMUX_PKG_DESCRIPTION="Bitcoin Core"
 TERMUX_PKG_LICENSE="MIT"
-TERMUX_PKG_VERSION=0.20.0
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=0.20.1
+TERMUX_PKG_REVISION=2
```

Comment:
A revision bump is for when the version wasn't changed, but we make some other change and still want to put out a new package. Whenever you bump the version, remove the revision, as it's no longer needed.